src/components/Grid.scss

```diff
@@ -60,12 +60,14 @@ $Grid-gutter: $su-medium;
 
 // * Modifier: gutters
 
-.Grid--withGutter {
-  margin: 0 (-0.5 * $Grid-gutter);
-}
-
 .Grid--withGutter > .Grid-cell {
   padding: 0 (0.5 * $Grid-gutter);
+  &:first-child {
+    padding-left: 0;
+  }
+  &:last-child {
+    padding-right: 0;
+  }
```

Comment:
Hrm.. how does this work? It's not just the first and last child we want to target.. right? 

I also have reservations about straying from the proven SUIT Grid pattern. The only reason it's broken is because of us adding this: https://github.com/teamsnap/teamsnap-ui/pull/55/commits/7c5a9c9fe627468415e8de4afc2f44573ae33bea#diff-f238cb697777bedff815cc431443ab2fR22

I think the better thing to do would be to remove that and address the instances where we need the Grid to be 100%.

Reply:
It wouldn't accommodate for grid cells dropping to new lines but assuming it's always..

```
<div class="Grid Grid--gridWithGutter">
  <div class="Grid-cell"></div>
  <div class="Grid-cell"></div>
  <div class="Grid-cell"></div>
</div>
```
..works fine

Didn't realize it was something added to LFE breaking it - yeah I'm cool with removing that width, and seeing what that might break

Reply:
Okay - I can take that on. We added it fix some of the full-width tables, but we'll have to find a work-around.

Reply:
Oh I see on the checkout, that wouldn't work for both checkout and overview usage.  Added a new width declaration with CSS calc() that seems to do the trick for both. Looks like it should cover our basis too : https://caniuse.com/#feat=calc

```
.Grid--withGutter {
  margin: 0 (-0.5 * $Grid-gutter);
  width: calc(100% + 16px); // 2
}
```

thoughts on that?